packages/database/src/cachers/EntityHierarchyCacher.js

```diff
@@ -20,6 +20,9 @@ export class EntityHierarchyCacher {
     this.changeHandlerCancellers[1] = this.models.entityRelation.addChangeHandler(
       this.handleEntityRelationChange,
     );
+    this.changeHandlerCancellers[2] = this.models.entityHierarchy.addChangeHandler(
+      this.handleEntityHierarchyChange,
+    );
```

Comment:
Although this is very readable, you can optionally turn it into a standard array assignment

```js
this.changeHandlerCancellers = [ ... ];
```

Reply:
Much more elegant!